tests/test_disassembly.py

```diff
@@ -38,6 +39,25 @@ def test_arm64_disect_instruction(self):
         regs_table = insn[insn.index("{") + 1 : insn.index("}")].replace(" ", "").split(",")
         assert ["v0.16b", "v1.16b", "v2.16b", "v3.16b"] == regs_table
 
+        proj = angr.load_shellcode(
+            b"\x00\xc0\x2d\xe9"
+            b"\x10\xf9\xf9\xe9",
+            archinfo.ArchARM(),
+            0,
+        )
+        # push {lr, pc}
+        # ldmib sb!, {r4, r8, fp, ip, sp, lr, pc}^
+
+        block = proj.factory.block(0)
+        disasm = proj.analyses[Disassembly].prep()(ranges=[(block.addr, block.addr + block.size)])
+        insns = [r for r in disasm.raw_result if isinstance(r, Instruction)]
+        rendered_insns = [i.render()[0].lower() for i in insns]
+        assert all(
+            i in rendered_insns[0] for i in ('{', '}', 'lr', 'pc')
+        )
+        assert 'sb!' in rendered_insns[1]
+        assert rendered_insns[1].endswith('^')
```

Comment:
Thanks for adding an ARM-specific test case. Can you please separate the AArch64 test case and this ARM test case into two methods?